### ISSUE TITLE
`img-uses-alt` rule reports false positive when using elements with dots

### ISSUE THREAD

**Issue by globexdesigns** [evcohen/eslint-plugin-jsx-a11y#1]
The following code:

```
<UX.Layout>
    test
</UX.Layout>
```

Reports the following error:

```
undefined elements must have an alt tag
```

It appears that when you use JSX components with a `.` in the name, it gets picked up by the rule.

**Comment by evcohen** [comment]
@globexdesigns thanks for reporting! Should be fixed with v0.4.2
